docs/architecture/015-cookies.md

```diff
@@ -0,0 +1,18 @@
+# Client Side Cookies
+
+## Context
+
+To read user's cookies, or not to read user cookies: that is the question. _Shakespeare_.
+
+Client side user cookies are an important part of web development architecture, but they contribute, from the server point of view, to impredictability in the rendering as seen and experienced by the user. In most cases the cookies are sent to the server as part of standard HTTP requests and the server can take care of generating the right output for the user, instad of postponing coookie driven user customization to the client side.
```

Comment:
Couple of typos
- impredictability -> unpredictability
- instad > instead

Reply:
Corrected. Thanks!

Reply:
Thanks. I've just noticed a triple-oh `coookie` in there too 😆